api/src/modules/authentication/get-user.decorator.ts

```diff
@@ -0,0 +1,9 @@
+import { createParamDecorator, ExecutionContext } from '@nestjs/common';
+import { User } from 'modules/users/user.entity';
+
+export const GetUser = createParamDecorator(
+  (_data, context: ExecutionContext): User => {
+    const request = context.switchToHttp().getRequest();
+    return request.user;
+  },
+);
```

Comment:
Since this is a global decorator, I think it should be added in
`api/src/decorators`